starboard/build/config/BUILDCONFIG.gn

```diff
@@ -451,6 +451,88 @@ template("shared_library") {
       }
     }
 
+    # Add an additional evergreen_loader target to every gtest target.
+    if (sb_is_evergreen_compatible && sb_evergreen_compatible_package &&
+        target_name != "loader_app") {
```

Comment:
Can you use `build_loader` here instead of referencing the target_name?

Reply:
Are we going to set build_loader to true for e.g. AOSP EG and Linux EG builder/github_config?

Reply:
`build_loader` is set per target, it's not a global GN flag, so we'd set it for the loader_app target and change the condition to instead be based on that here

Reply:
`build_loader` is a variable not directly tied to `build_with_separate_cobalt_toolchain`. It can be set per-target regardless of the toolchain and is only used in these templates

Reply:
We're going to wait until moving EG to a cobalt toolchain instead of it being its own platform